loader/trampoline.c

```diff
@@ -949,6 +964,7 @@ LOADER_EXPORT VKAPI_ATTR VkResult VKAPI_CALL vkAllocateMemory(VkDevice device, c
     return disp->AllocateMemory(device, pAllocateInfo, pAllocator, pMemory);
 }
 
+#ifndef VULKANSC
```

Comment:
Instead of all these ifdefs polluting the main file, why not create a non-sc trampoline that includes the files not used by SC.  Then you only have to ifdef that file?